test/test_serialization.py

```diff
@@ -377,11 +377,12 @@ def __reduce_ex__(self, proto):
 
         with tempfile.NamedTemporaryFile() as f:
             torch.save({"spoofed": TensorSerializationSpoofer(x)}, f)
-            f.seek(0)
-            with self.assertRaisesRegex(
-                    RuntimeError,
-                    "size is inconsistent with indices"):
-                y = torch.load(f)
+            for weights_only in (False, True):
```

Comment:
Can't use @parametrize here as this test is in `SerializationMixin` (doesn't inherit from TestCase)